File: BOJ/4344.cpp
#include<iostream>
using namespace std;

int main() {
    int C;
    cin >> C;

    for (int i = 0; i < C; i++) {
        int N;
        cin >> N;
        double* score = new double[N];
        double sum = 0;
        
        for (int j = 0; j < N; j++) {
            cin >> score[j];

            sum += score[j];
        }
        double avg;
        avg = sum / (double)N;
        
        int count = 0;
        for (int j = 0; j < N; j++) {
            if (avg < score[j]) {
                count++;
            }
        }

        // 소수점 아래 개수 ?
        cout.precision(3);
        cout << fixed;
        cout << count / (double)N * 100 << "%\n";

        delete[] score;
    }
}

File: BOJ/4344.py
c = int(input())

for _ in range(c) :
  data = list(map(int, input().split()))
  avg = sum(data[1:]) / data[0]
  count = 0

  for i in range(1, len(data)) :
    if data[i] > avg :
      count += 1

  result = count / data[0] * 100
  print('%.3f'% result+'%')